portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,34 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Hi everyone! My name is Jennifer Guerrero, and I am a sophomore at the University of Illinois at 
+          Urbana-Champaign. I am of Mexican/Honduran heritage, and I was born and raised in Chicago. I am a CSSI
+          alum, and I am very excited to learn more about CS through SPS and to meet new people.
+      </p>
+      <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+         <input type="submit" />
+      </form>
+      <div id= "hobbies">
+        <h2>Hobbies/Interests</h2>
+        <p>Click <a href="/plants.html">here</a> to see some quality green content. </p>
+      </div>
+      <p>Click here to get a random fact about me:</p>
```

Comment:
nit: serverHelloWorld() just prints Hello World not a random fact :)

Reply:
Changed in Jennifer_branch_week_2